# What we see in Hi-C: Introduction to chromatin biology and sequencing assays, Hi-C data browsing and feature extraction

Welcome to "What we see in Hi-C?" workshop!

This is a two-part intro to Hi-C data analysis: 

- Part I covers qualitative Hi-C data analysis in interactive browsers (higlass, resgen)
- Part II shows how to use flexibility of Python programming for whole-genome visualization, compartments, domains and scaling analysis of Hi-C with [Open2C tools](https://github.com/open2c).

Disclaimer: although we formally refer to whole-genome chromosome capture data as Hi-C, we will mostly work with Micro-C here. 

To start, follow the instructions in [What_we_see_in_Hi-C notebook](https://github.com/mirnylab/Hi-C_Workshop/blob/main/What_we_see_in_Hi-C_3June2025.ipynb). 

Author: Aleksandra Galitsyna, based on the materials from [Open2C](https://github.com/open2c) and [4DN](https://www.4dnucleome.org/software/) communities. 

## Note on releases:

First version of the workshop has been presented at the conference "Genome Architecture and Function" in Boston, 2024. 
- Workshop 2024 link: https://github.com/mirnylab/Hi-C_Workshop/releases/tag/v1.0
- Workshop 2024 recording: https://youtu.be/Z7fa5_xfifc

Next, the workshop waspresented at the conference "Physics Meets Genome" in Cargese, 2025.
